Real Analysis/real-analysis-notes.tex

```diff
@@ -142,4 +146,263 @@ \subsection*{(1/19/2017) Lecture 1}
+      \pi_\alpha^{-1}(E_\alpha) = \prod_{\beta \in A} F_\beta
+    \]
+    where $F_\beta = E_\alpha$ if $\beta = \alpha$ and $F_\beta =
+    X_\alpha$ if $\beta \neq \alpha$. Thus, $\Ep_1 \subset \Ep_2$
+
+  \item ($\supset$) It is enough to show that $\Ep_2 \subset
+    \M(\Ep_1)$. Let $Y = \prod_{\alpha \in A} E_\alpha \in \Ep_2$
+    where, for all $\alpha \in A$, $E_\alpha \in \M_\alpha$. Because
+    $A$ is at most countable, we have that \[
+      Y = \bigcap_{\alpha \in A} \pi_\alpha^{-1}(E_\alpha).
+    \]
+    So, if $x = (x_\alpha)_{\alpha \in Y}$, then $x \in
+    \pi_\alpha^{-1}(E_\alpha) (\in \Ep_1) \ \iff \ x_\alpha \in
+    E_\alpha$. Thus, $Y \in \M(\Ep_1)$ because $Y$ is (at most) countable
+    intersection of elements of $\M(\Ep_1)$ which is a $\sigma$-algebra.
+  \end{itemize}
+\end{proof}
+\begin{prop}
+  Suppose we have $(x_\alpha)_{\alpha \in A}$ and $\Ep_\alpha \subset
+  \P(X_\alpha)$. Let $\M_\alpha = \M(\Ep_\alpha)$ in $X_\alpha$. Then
+  \begin{itemize}
+  \item $\bigoplus_{\alpha \in A} \M_\alpha = \M(\F_1)$ where $\F_1 =
+    \{\pi_\alpha^{-1}(E_\alpha) \mid \alpha \in X, E_\alpha \in \Ep_\alpha\}$
+  \item If, in addition, $A$ is at most countable, then \[
+      \bigoplus_{\alpha \in A} = \M(\F_2)
+    \]
+    with $\F_2 = \{\prod_{\alpha \in A} E_\alpha \mid \forall \alpha
+    \in A, E_\alpha \in \Ep_\alpha\}$
+  \end{itemize}
+\end{prop}
+\begin{proof}
+  The proof of this proposition is omitted. One can refer to
+  proposition 1.4 on page 23 of Folland to get an idea of how to prove this.
+\end{proof}
+\begin{prop}
+  Let $X_1, \ldots, X_n$ be metric spaces and \[
+    X = \prod_{j=1}^n X_j.
+  \]
+  Then,
+  \begin{enumerate}
+  \item $\B_{X_j} \subset \B_X$.
+  \item If, in addition, the $X_j$'s are seperable, then \[
+      \bigotimes_{j=1}^n \B_{X_j} = \B_X.
+    \]
+    Recall that a space $Y$ is separable if and only if $Y$ has a
+    countable dense subset.
+  \end{enumerate}
+\end{prop}
+\begin{cor}
+  \[
+    \bigotimes_{j=1}^n \B_\R = \B_{\R^n}
+  \]
+\end{cor}
+\begin{proof}[Proof of Corollary]
+  $\R$ is seperable.
+\end{proof}
+\begin{proof}[Proof of Proposiiton]
+  The proof of this proposition is also ommitted but can be found
+  under the proof of Proposition 1.5 on page 23 of Folland.
+\end{proof}
+\subsection*{(1/24/2017) Lecture 3}
+\subsection{Point-Set Topology}
+\begin{defn}
+  Let $X$ be a set. A \emph{topology} on $X$ is a set of ``open sets''
+  $\Top \in \P(X)$ such that
+  \begin{itemize}
+  \item $\varnothing, X \in \Top$.
+  \item $\Top$ is stable under finite intersections.
+  \item $\Top$ is stable by arbitrary unions.
+  \end{itemize}
+  We call the pair $(X,\Top)$ a \emph{topological space}.
+\end{defn}
+\begin{rmk}
+  If $(X,d)$ is a metric space, there exists a canonically define
+  topology, $\Top(d)$, given by the characterization that \[
+    U \in \Top(d) \ \iff \ \forall x \in U, \exists r > 0 \text{ such
+      that } B(x,r) \subset U.
+  \]
+\end{rmk}
```

Comment:
I don't understand the remark about "equivalent." A topology is just a subset of the power set of X. If two topologies are equivalent if they are equivalent as sets.